third_party/LiteLLM/README.md

```diff
@@ -0,0 +1,159 @@
+# Use LiteLLM Proxy to Call Mistral AI API
+
+Use [LiteLLM Proxy](https://docs.litellm.ai/docs/simple_proxy) for:
+- Calling 100+ LLMs Mistral AI, OpenAI, Azure, Vertex, Bedrock/etc. in the OpenAI ChatCompletions & Completions format
+- Track usage + set budgets with Virtual Keys
+
+Works for [Mistral AI API](https://docs.litellm.ai/docs/providers/mistral) + [Codestral API](https://docs.litellm.ai/docs/providers/codestral) + [Bedrock](https://docs.litellm.ai/docs/providers/bedrock)
+
+## Sample Usage
+
+### Step 1. Create a Config for LiteLLM proxy
+
+LiteLLM Requires a config with all your models define - we can call this file `litellm_config.yaml`
+
+[Detailed docs on how to setup litellm config - here](https://docs.litellm.ai/docs/proxy/configs)
+
+```yaml
+model_list:
+  - model_name: mistral-small-latest ### MODEL Alias ###
+    litellm_params: # all params accepted by litellm.completion() - https://docs.litellm.ai/docs/completion/input
+      model: mistral/mistral-small-latest ### MODEL NAME sent to `litellm.completion()` ###
+      api_key: "os.environ/MISTRAL_API_KEY" # does os.getenv("MISTRAL_API_KEY")
+  - model_name: mistral-nemo
+    litellm_params: 
+      model: mistral/open-mistral-nemo 
+      api_key: "os.environ/MISTRAL_API_KEY"
+
+```
+
+### Step 2. Start litellm proxy
+
+```shell
+docker run \
+    -v $(pwd)/litellm_config.yaml:/app/config.yaml \
+    -e MISTRAL_API_KEY=<your-mistral-api-key>
+    -p 4000:4000 \
+    ghcr.io/berriai/litellm:main-latest \
+    --config /app/config.yaml --detailed_debug
+```
+
+### Step 3. Test it! 
+
+[Use with Langchain, LlamaIndex, Instructor, etc.](https://docs.litellm.ai/docs/proxy/user_keys)
+
+```bash
+import openai
```

Comment:
Could you use Mistral SDK instead of OpenAI please?

Reply:
Updated @sophiamyang - now examples use the Mistral SDK